WickedEngine/wiJobSystem.cpp

```diff
@@ -73,13 +86,13 @@ namespace wiJobSystem
 		{
 			std::thread worker([] {
 
-				while (true)
+				while (jobs_alive)
```

Comment:
Isn't this thread unsafe?

Reply:
Yes, it should be atomic for one, and not be destroyed until any workers are alive. Probably need a better approach.